azure-automation-plus-dsc-lab/nested/15.21.00.createWebServers.json

```diff
@@ -0,0 +1,145 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "webPrefix": {
+      "type": "string"
+    },
+    "azureRegion": {
+      "type": "string"
+    },
+    "adminUserName": {
+      "type": "string"
+    },
+    "adminPassword": {
+      "type": "securestring"
+    },
+    "webAvSetId": {
+      "type": "string"
+    },
+    "webNicIds": {
+      "type": "array"
+    },
+    "saName": {
+      "type": "string"
+    },
+    "domainName": {
+      "type": "string"
+    },
+    "domainJoinOptions": {
+      "type": "string"
+    }
+  },
+  "variables": {
+    "web01name": "[concat(parameters('webPrefix'),1)]",
+    "webVmSize": "Standard_A1_v2",
+    "webServerInstances": 2,
+    "imagePublisher": "MicrosoftWindowsServer",
+    "imageOffer": "WindowsServer",
+    "imageSku": "2016-Datacenter-Server-Core-smalldisk",
+    "diskNameOs": "[toLower(concat(variables('web01name'),'-syst'))]",
+    "saType": "Standard_LRS",
+    "diskNameData": "[toLower(concat(variables('web01name'),'-data'))]",
+    "saUrlSuffix": ".blob.core.windows.net",
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#referencing-resource-properties) for how to handle storage account uris